services/ec2/src/main/java/org/finra/gatekeeper/configuration/properties/GatekeeperEC2Properties.java

```diff
@@ -29,6 +27,12 @@
      */
     private String appIdentityTag;
 
+    private boolean explanationFieldRequired;
+
+    private boolean ticketIdFieldRequired;
+
+    private String ticketIdFieldMessage;
+
```

Comment:
This should be handled in the Common area, that way it only has to be set once by the user